content/webapp/components/SectionHeader/SectionHeader.tsx

```diff
@@ -34,17 +35,23 @@ const Title = styled.span`
 
 type Props = {
   title: string;
+  isGridSize12?: boolean;
```

Comment:
I suppose you could have this take the `gridSize` function as a param to avoid the boolean? That might be even more re-usable?

Reply:
I like this yeah!

Reply:
Updated [here](https://github.com/wellcomecollection/wellcomecollection.org/pull/11062/commits/b4b5fa473b72d066ce86cdb528d9c24a3b05a992)